fn outer() {
  raiser();
}

fn raiser() {
  raise Error("raise");
}

try {
  outer();
  assert(false);
} catch {
  assert(true);
}